src/test/java/com/codeborne/selenide/collections/ExactTextsTest.java

```diff
@@ -98,6 +99,28 @@ void failOnTextMismatch() {
     }
   }
 
+  @Test
```

Comment:
Also since we are using AssertJ you can use [assertThatThrownBy()](https://joel-costigliola.github.io/assertj/assertj-core-features-highlight.html#exception-assertion) assertions for exceptions. You can find example in our existing tests.

Reply:
fixed 
```assertThatThrownBy(() -> exactTexts.fail(mockedElementsCollection,
      singletonList(mockedWebElement), exception, 10000))
      .isInstanceOf(TextsSizeMismatch.class)
      .hasMessageContaining("Actual: [One, Two], List size: 1")
      .hasMessageContaining("Expected: [One, Two], List size: 2")
      .hasMessageEndingWith("Collection: Collection description");
```